### PR TITLE
feat(flags): Auto rollback beta

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

Hackathon from team experiments october offsite. Key motivation here is based on long term vision discussion we want feature flags to have more automated functionality. As a starter for our hackathon, we've decided to create an auto rollback trigger based on sentry metrics or insight metrics. This is a beta rollout for internal testing

Relevant links:
Long term vision: https://github.com/PostHog/posthog.com/pull/4516
Related RFC for extension: https://github.com/PostHog/product-internal/issues/369

## Changes
- add ability to auto rollback feature flags based on sentry errors or insight changes
- insight metric is based on 7 day rolling average
- sentry metric is based on a simple threshold of error count
- sentry metric condition is only usable if a sentry api key is provided in the instance settings of the organization

Screen Shot 2022-11-01 at 1 46 16 PM

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Comment by liyiy** [comment]
My only nit is that you can't see this on the feature flag's "view mode" so you'll have to go into "edit" mode just to see what auto rollback conditions you've set up. We don't really need anything fancy but a simple display could help a lot here? 

Also there's a permanent spinner thing happening when I check the auto rollback conditions after going into "edit" mode

Screen Shot 2022-11-01 at 2 02 38 PM

Also do you mind filling in the summary a bit more about what this PR came out of (team offsite hackathon), and what its initial merge is for (internal testing), and adding any relevant links? 😁

**Comment by EDsCODE** [comment]
@liyiy here's the view. The loader should complete also now

Last thing of note is how to show an event/action with filters. 
Screen Shot 2022-11-01 at 4 11 46 PM

**Review by liyiy — APPROVED** [review]
LGTM! Final nit is whether you want to change the name of the migration to something more readable like "add feature flags auto rollback" instead of "auto4853475934857" but it's fine either way 👍

File: posthog/migrations/0275_auto_20221027_1928.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.15 on 2022-10-27 19:28
```
Comment (neilkakkar) [inline posthog/migrations/0275_auto_20221027_1928.py:1]: rename?

**Review by neilkakkar — APPROVED** [review]

**Pull request merged**